pyspectral/tests/test_rayleigh.py

```diff
@@ -163,6 +164,44 @@ def test_rayleigh_init(self, download_luts, exists):
             this = rayleigh.Rayleigh('NOAA-19', 'avhrr/3', atmosphere='tropical')
             self.assertTrue(this.sensor == 'avhrr3')
 
+    @patch('pyspectral.rayleigh.HAVE_DASK', False)
+    @patch('os.path.exists')
+    @patch('pyspectral.utils.download_luts')
+    @patch('pyspectral.rayleigh.get_reflectance_lut_from_file')
+    @patch('pyspectral.rsr_reader.RelativeSpectralResponse._get_rsr_data_version')
+    @patch('pyspectral.rayleigh.Rayleigh.get_effective_wavelength')
+    def test_get_reflectance_redband_outside_clip(self, get_effective_wvl,
+                                                  get_rsr_version, get_reflectance_lut_from_file,
+                                                  download_luts, exists):
+        """Test getting the reflectance correction - using red band reflections outside 0-100."""
+        rayl = TEST_RAYLEIGH_LUT
+        wvl_coord = TEST_RAYLEIGH_WVL_COORD
+        azid_coord = TEST_RAYLEIGH_AZID_COORD
+        sunz_sec_coord = TEST_RAYLEIGH_SUNZ_COORD
+        satz_sec_coord = TEST_RAYLEIGH_SATZ_COORD
+
+        get_reflectance_lut_from_file.return_value = (rayl, wvl_coord, azid_coord,
```

Comment:
Use the global constants directly here, no need to lengthen the test by using additional variables.

Reply:
@pnuu In the mean time I have collected everything in `setUp`, so it should be a lot better now at least.